.github/CODEOWNERS

```diff
@@ -1,16 +1,15 @@
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
-*                  @Sage/carbon
+*                        @Sage/carbon
 
 # The Carbon Dev team are responsible for reviewing Pull Requests
 # that contain any change modifying JS, JSX, TS or TSX files
-*.js               @Sage/carbon-dev
-*.ts               @Sage/carbon-dev
+*.js                     @Sage/carbon-dev
+*.ts                     @Sage/carbon-dev
 
 # The Carbon QA team are responsible for reviewing Pull Requests
 # that contain any change modifying spec and test.js files 
```

Comment:
Suggestion(minor): Remove the word spec now

Could also do with updating what was test.js as well since that has seemingly changed as times gone by